pkg/tsdb/schema/schema.go

```diff
@@ -56,12 +56,16 @@ func newSchema(samplesIngestionRate, aggregationGranularity, aggregatesList stri
 		return nil, errors.New("Cross label aggregations must be used in conjunction with aggregations.")
 	}
 
+	if len(aggregates) == 0 {
+		aggregates = strings.Split(config.DefaultAggregates, ",")
+	}
+
 	defaultRollup := config.Rollup{
-		Aggregates:             []string{},
+		Aggregates:             aggregates,
 		AggregationGranularity: aggregationGranularity,
 		StorageClass:           config.DefaultStorageClass,
-		SampleRetention:        sampleRetention,                  //TODO: make configurable
-		LayerRetentionTime:     config.DefaultLayerRetentionTime, //TODO: make configurable
+		SampleRetention:        sampleRetention, //TODO: make configurable
```

Comment:
what about this one? (TODO)

Reply:
CI should pass, please fix the CI

Reply:
Is not related here, will do someday in a different PR.